tests/10-verify.spec.js

```diff
@@ -251,144 +272,110 @@ describe('verify API', () => {
   });
 });
 
-function _generateKeyId({did, key}) {
-  // `did` + multibase base58 (0x7a / z) encoding + key fingerprint
-  return `${did}#z${key.fingerprint()}`;
-}
+describe('verify API (presentations', () => {
```

Comment:
is this supposed to be `verify API (presentations)`?

Reply:
Ah yeah, thanks.